Tutorial/Consuming packages/Build a simple CMake project using Conan wrong command for windows

# Error
```
PS C:\Users\FreePhoenix\Documents\Programming\examples2\tutorial\consuming_packages\simple_cmake_project\build> cmake .. -G "Visual Studio 17 2022" -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake
CMake Warning:
  Ignoring extra path from command line:

   ".cmake"


CMake Error at C:/Soft/CMake/share/cmake-3.28/Modules/CMakeDetermineSystem.cmake:176 (message):
  Could not find toolchain file: conan_toolchain
Call Stack (most recent call first):
  CMakeLists.txt:2 (project)


-- Configuring incomplete, errors occurred!
```

--- a/tutorial/consuming_packages/build_simple_cmake_project.rst
+++ b/tutorial/consuming_packages/build_simple_cmake_project.rst
@@ -241,7 +241,7 @@ Now we are ready to build and run our **compressor** app:
 
     $ cd build
     # assuming Visual Studio 15 2017 is your VS version and that it matches your default profile
-    $ cmake .. -G "Visual Studio 15 2017" -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake
+    $ cmake .. -G "Visual Studio 15 2017" -DCMAKE_TOOLCHAIN_FILE="conan_toolchain.cmake"
     $ cmake --build . --config Release
     ...
     [100%] Built target compressor

